integration-tests/src/test/resources/domain-home-on-pv/create-domain.py

```diff
@@ -48,26 +48,158 @@ def getEnvVar(var):
 # Configure the Administration Server
 # ===================================
 cd('/Servers/AdminServer')
-# Dont set listenaddress, introspector overrides automatically with sit-config
-#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
 set('ListenPort', admin_port)
 set('Name', admin_server_name)
 
 create('T3Channel', 'NetworkAccessPoint')
 cd('/Servers/%s/NetworkAccessPoints/T3Channel' % admin_server_name)
 set('PublicPort', t3_channel_port)
-set('PublicAddress', 'junkvalue')
-# Dont set listenaddress, introspector overrides automatically with sit-config
-#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+set('PublicAddress', t3_public_address)
+set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
 set('ListenPort', t3_channel_port)
 
-cd('/Servers/%s' % admin_server_name)
-create(admin_server_name,'Log')
-cd('/Servers/%s/Log/%s' % (admin_server_name, admin_server_name))
-# Give incorrect filelog, introspector overrides with sit-config
-set('FileName', 'dirdoesnotexist')
+# Set the admin user's username and password
+# ==========================================
+cd('/Security/%s/User/weblogic' % domain_name)
+cmo.setName(admin_username)
+cmo.setPassword(admin_password)
+
+# Write the domain and close the domain template
+# ==============================================
+setOption('OverwriteDomain', 'true')
+
+# Create a cluster
+# ======================
+cd('/')
+cl=create(cluster_name, 'Cluster')
+
+if cluster_type == "CONFIGURED":
+
+  # Create managed servers
+  for index in range(0, number_of_ms):
+    cd('/')
+
+    msIndex = index+1
+    name = '%s%s' % (managed_server_name_base, msIndex)
+    name_svc = '%s%s' % (managed_server_name_base_svc, msIndex)
+
+    create(name, 'Server')
+    cd('/Servers/%s/' % name )
+    print('managed server name is %s' % name);
+    set('ListenAddress', '%s-%s' % (domain_uid, name_svc))
+    set('ListenPort', server_port)
+    set('NumOfRetriesBeforeMSIMode', 0)
+    set('RetryIntervalBeforeMSIMode', 1)
+    set('Cluster', cluster_name)
+
+else:
+  print('Configuring Dynamic Cluster %s' % cluster_name)
+
+  templateName = cluster_name + "-template"
+  print('Creating Server Template: %s' % templateName)
+  st1=create(templateName, 'ServerTemplate')
+  print('Done creating Server Template: %s' % templateName)
+  cd('/ServerTemplates/%s' % templateName)
+  cmo.setListenPort(server_port)
+  cmo.setListenAddress('%s-%s${id}' % (domain_uid, managed_server_name_base_svc))
```

Comment:
We should not be setting listen-addresses to their exact expected value, as automatic situational config is always generated and always expected to override the value, and we need to verify that the automatic situational config is working. A public-address, on the other hand, requires 'manually' setting to the correct value if it's different than the listen-address - this can be done in the original configuration or via 'custom overrides'.